projects/tinygltf/build.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash -eu
+# Copyright (C) 2023 ISP RAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+export MAIN_OBJ=""
+
+if [[ $CONFIG = "libfuzzer" ]]
+then
+    export CC="clang"
+    export CXX="clang++"
+    export CFLAGS="-g -O0 -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
+    export CXXFLAGS="-g -std=c++11 -O0 -fsanitize=fuzzer-no-link,address,integer,bounds,null,undefined,float-divide-by-zero"
```

Comment:
do we need to compile with `-O0` optimization level? here and later

Reply:
this option was in project's makefile build